libs/platform/core/src/services/page-meta/default-page-meta.service.ts

```diff
@@ -93,10 +93,15 @@ export class DefaultPageMetaService implements PageMetaService {
         continue;
       }
 
-      attrs += `[${key}="${value}"]`;
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      attrs += `[${key}="${this.escapeValue(value!)}"]`;
     }
 
     const name = this.getTagName(definition.name);
     return document.head.querySelector(`${name}${attrs}`);
   }
+
+  protected escapeValue(value: string): string {
+    return value.replace(/"/g, '\\$&');
```

Comment:
Can we cover this with a test? 
And perhaps we test other characters that could break it?

Reply:
I believe we should replace it with `&quot;` as other wise search engines will break, see [https://support.sitebeat.com/help/what-is-a-page-meta-title-description and https://stackoverflow.com/questions/1137045/meta-description-quotes-escape

Reply:
Only quotes might break it, cause we create selector `key="${value}"